easybuild/easyconfigs/r/rmats-turbo/rmats-turbo-4.1.1-foss-2020b.eb

```diff
@@ -0,0 +1,64 @@
+easyblock = 'Bundle'
+
+name = 'rmats-turbo'
+version = '4.1.1'
+
+homepage = 'https://github.com/Xinglab/rmats-turbo'
+description = """rMATS turbo is the C/Cython version of rMATS (refer to http://rnaseq-mats.sourceforge.net)."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+
+builddependencies = [
+    ('CMake', '3.18.4'),
+]
+dependencies = [
+    ('Python', '3.8.6'),
+    ('GSL', '2.6'),
+    ('SAMtools', '1.12'),
+    ('STAR', '2.7.7a'),
+    ('BamTools', '2.5.1'),
+]
+
+local_ldflags = " -lm -lgfortran -lgsl -lgslcblas -lgomp -lopenblas $LIBLAPACK "
+local_buildopts = '  CC="$CC" CXX="$CXX" FC="$FC" LDFLAGS="%s"' % local_ldflags
+
+default_component_specs = {
+    'sources': ['v%(version)s.tar.gz'],
+    'source_urls': ['https://github.com/Xinglab/%(name)s/archive/refs/tags/'],
+    'checksums': ['cdfde24b68be0b41b4ca43e5414d3c292deb7efd617d7ee2315e2d6e6bcb1c92'],
+}
+
+components = [
+    (name, version, {
+        'easyblock': 'MakeCp',
+        'start_dir': 'rmats-turbo-%s/rMATS_C' % version,
+        'buildopts': "%s" % local_buildopts,
+        'files_to_copy': [(['rMATSexe', '../rmats.py'], 'bin')],
+    }),
+    ('rmats-turbo-python', version, {
+        'easyblock': 'PythonPackage',
+        'patches': [('%s-%%(version)s_bamtools_as_dependency.patch' % name)],
```

Comment:
I wouldn't use `name` here at all, just hardcode (since the name is unlikely to change):

```python
        'patches': [('rmath-turbo-%(version)s_bamtools_as_dependency.patch')],
```